packages/synapse-interface/tailwind.config.js

```diff
@@ -113,12 +113,13 @@ module.exports = {
         // updated colors
         strong: '#FCFCFD',
         primary: '#EEEDEF',
-        secondary: '#C0BCC2',
+        secondary: '#BFBCC2',
```

Comment:
Figma still has the `--secondary` color listed as #C0BCC2, is #BFBCC2 supposed to be another variant of the `--secondary` color?